sbnci/Common/scripts/tpcsimvalidation.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 
+# Set code name and version to have experiment agnostic env vars
+source sbnci_setcodename.sh
+
 ## Introduce environment variables specific to this validation
-export ref_sbndcode_ana_hist=${ref_tpcsimvalidation_hist}
+export ref_ana_hist=${ref_tpcsim_hist}
 export normalise_plots=${normalise_plots_tpcsimvalidation}
 
 ## Name of plotting script. This should be the only line that needs changing for other CI chains.
-plotScript="$SBNCI_DIR/scripts/sbnciplot_tpcsimvalidation.C"
+plotScript="$SBNCI_DIR/scripts/sbnciplot_tpcsimvalidation_${expName}.C"
```

Comment:
Likewise should this be?
```suggestion
plotScript="$expCIDir/PlottingScripts/sbnciplot_tpcsimvalidation_${expName}.C"
```

This is then true across all the validation.sh scripts!